chirp/drivers/kguv980p.py

```diff
@@ -0,0 +1,2722 @@
+            # pwr_index= mem.power
+            if str(mem.power) == "None":
+                mem.power = self.POWER_LEVELS[1]
+            index = self.POWER_LEVELS.index(mem.power)
+            if index == 2:
+                _mem.power = 0b11
+            else:
+                _mem.power = self.POWER_LEVELS.index(mem.power)
+            # Not sure what this bit does yet but it causes
+            # the radio to display
+            # MED power when the CPS shows Low Power.
+            # Forcing it to 0 to keep them
+            # consistent
+            _mem.extra_power_bit = 0
+            # Set other unknowns to 0 to match default CPS values
+            _mem.unknown1 = 0
+            _mem.unknown2 = 0
+            _mem.unknownbit2 = 0
+            _mem.unknown3 = 0
+            _mem.Unknown4_shft_dir = 0
+            _mem.unknown5 = 0
+
+            #  set to mute mode to QT (not QT+DTMF or QT*DTMF) by default
+            #  This changes them in the channel memory
+            _mem.mute_mode = 1
+
+    def _get_settings(self):
+        _settings = self._memobj.settings
+        _limits = self._memobj.bandlimits
+        _oem = self._memobj.oem
+
+        cfg_grp = RadioSettingGroup("cfg_grp", "Config Settings")
+        cfg1_grp = RadioSettingGroup("cfg1_grp", "Config Settings 1")
+        cfg2_grp = RadioSettingGroup("cfg2_grp", "Config Settings 2")
+        if self.MODEL == "KG-1000G Plus":
+            vfoa_grp = RadioSettingGroup("vfoa_grp", "Area A Settings")
+        else:
+            vfoa_grp = RadioSettingGroup("vfoa_grp", "VFO A Settings")
+        vfo150_grp = RadioSettingGroup("vfo150_grp", "150M Settings")
+        vfo450_grp = RadioSettingGroup("vfo450_grp", "450M Settings")
+        vfo20_grp = RadioSettingGroup("vfo20_grp", "20M Settings")
+        vfo50_grp = RadioSettingGroup("vfo50_grp", "50M Settings")
+        vfo350_grp = RadioSettingGroup("vfo350_grp", "350M Settings")
+        vfo850_grp = RadioSettingGroup("vfo850_grp", "850M Settings")
+        if self.MODEL == "KG-1000G Plus":
+            vfob_grp = RadioSettingGroup("vfob_grp", "Area B Settings")
+        else:
+            vfob_grp = RadioSettingGroup("vfob_grp", "VFO B Settings")
+
+        fmradio_grp = RadioSettingGroup("fmradio_grp", "FM Broadcast Memory")
+        lmt_grp = RadioSettingGroup("lmt_grp", "Frequency Limits")
+        lmwrn_grp = RadioSettingGroup("lmwrn_grp", "USE AT YOUR OWN RISK")
+        rxlim_grp = RadioSettingGroup("rxlim_grp", "Rx Limits")
+        txlim_grp = RadioSettingGroup("txlim_grp", "Tx Limits")
+        oem_grp = RadioSettingGroup("oem_grp", "OEM Info")
+        scan_grp = RadioSettingGroup("scan_grp", "Scan Group")
+        scanname_grp = RadioSettingGroup("scanname_grp", "Scan Names")
+        remote_grp = RadioSettingGroup("remote_grp", "Remote Settings")
+        extra_grp = RadioSettingGroup("extra_grp",
+                                      "Extra Settings")
+        if self.MODEL == "KG-1000G Plus":
```

Comment:
Your suggestion makes sense and is consistent with what I did elsewhere.   The distributors of the radio would like to have the settings names match the CPS and Manuals since there was a pretty definitive migration in terms from a HAM radio to GMRS and they changed the radio menu items to use more GMRS friendly terms.

Reply:
Ack, yeah, I figure. Anyway, this was not a critical thing but it's cleaner and like you say, matches what you did elsewhere, so cool.